fullstop-plugins/fullstop-save-securitygroups-plugin/src/main/java/org/zalando/stups/fullstop/plugin/SaveSecurityGroupsPlugin.java

```diff
@@ -122,21 +123,18 @@ public void processEvent(final CloudTrailEvent event) {
             String instanceBucketNameControlElement = null;
             DateTime instanceBootTimeControlElement = null;
 
+            // TODO, I do not understand what is going on here and why
```

Comment:
find out wich bucket starts with our instance-id. so we cann  add the file in the right directory